test/fixtures/api_resources.yml

```diff
@@ -46,6 +46,19 @@ monitoring_target:
                 "last_checked_at": "2022-04-23T23:02:34.592+00:00",
                 "log_incidents_to": "monitoring-target-incident"
               }
+tls_monitoring_target:
```

Comment:
can these dependents be moved to their own separate files? We want to prevent the model fixture file from getting overcrowded with plugin code